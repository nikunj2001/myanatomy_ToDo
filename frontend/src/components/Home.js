import React, { useEffect, useState } from 'react'
import CreateTask from './CreateTask';
import Nav from './Nav';
import ViewTask from './ViewTask';

const Home = () => {
  return (
    <>
        <CreateTask/>
        <ViewTask/>
    </>
  )
}

export default Home;