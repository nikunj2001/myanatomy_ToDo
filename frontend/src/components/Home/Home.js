import React, { useEffect, useState } from 'react'
import CreateTask from '../CreateTask';
import Nav from '../Nav';
import ViewTask from '../ViewTask/ViewTask';

const Home = () => {
  return (
    <div>
      <CreateTask />
      <ViewTask />
    </div>
  )
}

export default Home;