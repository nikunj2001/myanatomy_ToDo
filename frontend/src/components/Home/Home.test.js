import React from 'react';
import { shallow, configure } from "enzyme";
import Enzyme from 'enzyme';
import Adapter from "@wojtekmaj/enzyme-adapter-react-17"
import Home from './Home';

Enzyme.configure({ adapter: new Adapter() });


describe("Home", () => {
    it("should render my component", () => {
        const wrapper = shallow(<Home />);
        const tree = wrapper.debug();
        expect(tree).toMatchSnapshot();
    });

});