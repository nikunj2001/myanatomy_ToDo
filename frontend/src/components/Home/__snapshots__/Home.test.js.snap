// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should render my component 1`] = `
"<div>
  <CreateTask />
  <ViewTask />
</div>"
`;
